tests/production/AddUserSettingsToCouchTests.js

```diff
@@ -0,0 +1,122 @@
+/**
+Tests for adding test gpii keys and 'user' prefs safe.
+
+Copyright 2019 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+
+"use strict";
+
+var fluid = require("infusion"),
+    gpii = fluid.registerNamespace("gpii");
+
+fluid.require("%gpii-universal");
+
+gpii.loadTestingSupport();
+
+fluid.registerNamespace("gpii.tests.productionConfigTesting");
+
+require("./Common.js");
```

Comment:
We've been using file name convention containing "utils" for holding shared utility functions and grades. Can you rename "Common.js" to names like "ProductionTestsUtils.js"? Thanks.

Reply:
Sure, but I'll have to check that all imports of "Common.js" are properly updated.  Similarly, renaming of the disruption sequence grade itself and "DisruptionSequenceGradeConfig.js" will affect everywhere they are used.